src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +44,32 @@
      * Андрей - 600
      * София - 900</p>
      */
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
let's make it private, since we don't need it outside of this class

Reply:
Ok, I did.